src/ImageSharp/Formats/Webp/Lossy/Vp8Encoding.cs

```diff
@@ -65,48 +65,16 @@ internal static unsafe class Vp8Encoding
 
         public static readonly int[] Vp8I4ModeOffsets = { I4DC4, I4TM4, I4VE4, I4HE4, I4RD4, I4VR4, I4LD4, I4VL4, I4HD4, I4HU4 };
 
-#if SUPPORTS_RUNTIME_INTRINSICS
-#pragma warning disable SA1310 // Field names should not contain underscore
-        private static readonly Vector128<short> K1 = Vector128.Create((short)20091).AsInt16();
-
-        private static readonly Vector128<short> K2 = Vector128.Create((short)-30068).AsInt16();
-
-        private static readonly Vector128<short> Four = Vector128.Create((short)4);
-
-        private static readonly Vector128<short> Seven = Vector128.Create((short)7);
-
-        private static readonly Vector128<short> K88p = Vector128.Create(8, 0, 8, 0, 8, 0, 8, 0, 8, 0, 8, 0, 8, 0, 8, 0).AsInt16();
-
-        private static readonly Vector128<short> K88m = Vector128.Create(8, 0, 248, 255, 8, 0, 248, 255, 8, 0, 248, 255, 8, 0, 248, 255).AsInt16();
-
-        private static readonly Vector128<short> K5352_2217p = Vector128.Create(232, 20, 169, 8, 232, 20, 169, 8, 232, 20, 169, 8, 232, 20, 169, 8).AsInt16();
-
-        private static readonly Vector128<short> K5352_2217m = Vector128.Create(169, 8, 24, 235, 169, 8, 24, 235, 169, 8, 24, 235, 169, 8, 24, 235).AsInt16();
-
-        private static readonly Vector128<int> K937 = Vector128.Create(937);
-
-        private static readonly Vector128<int> K1812 = Vector128.Create(1812);
-
-        private static readonly Vector128<short> K5352_2217 = Vector128.Create(169, 8, 232, 20, 169, 8, 232, 20, 169, 8, 232, 20, 169, 8, 232, 20).AsInt16();
-
-        private static readonly Vector128<short> K2217_5352 = Vector128.Create(24, 235, 169, 8, 24, 235, 169, 8, 24, 235, 169, 8, 24, 235, 169, 8).AsInt16();
-
-        private static readonly Vector128<int> K12000PlusOne = Vector128.Create(12000 + (1 << 16));
-
-        private static readonly Vector128<int> K51000 = Vector128.Create(51000);
-
-        private static readonly byte MmShuffle2301 = SimdUtils.Shuffle.MmShuffle(2, 3, 0, 1);
-
-        private static readonly byte MmShuffle1032 = SimdUtils.Shuffle.MmShuffle(1, 0, 3, 2);
-#pragma warning restore SA1310 // Field names should not contain underscore
-#endif
-
-        static Vp8Encoding()
+        private static byte[] GetClip1()
```

Comment:
Instead of the static constructor (cctor) this method is now called from the `static readonly` field.
Avoiding a cctor and using `beforefieldinit` (IL) is the prefered way in dotnet/runtime too, as there are several pro reasons for it, liking timing, static init check, etc.

That's the only other code change not mentioned in PR's description.